src/promptflow-evals/promptflow/evals/evaluate/_batch_run_client/batch_run_context.py

```diff
@@ -26,6 +31,12 @@ def __enter__(self):
             os.environ[PF_FLOW_ENTRY_IN_TMP] = "true"
             os.environ[PF_FLOW_META_LOAD_IN_SUBPROCESS] = "false"
 
+            if os.environ.get(PF_BATCH_TIMEOUT_SEC) is None:
+                os.environ[PF_BATCH_TIMEOUT_SEC] = str(PF_BATCH_TIMEOUT_SEC_DEFAULT)
+                os.environ[PF_IS_BATCH_TIMEOUT_SYSTEM_SET] = "true"
```

Comment:
Why not use a class member variable of boolean type instead?

Reply:
done